examples/cp/atomic_reference_example.py

```diff
@@ -0,0 +1,17 @@
+import hazelcast
```

Comment:
nit: does it make sense to add an org web sample for AtomicReference?

Reply:
I thought about adding it but I saw that we don't have the sample for the AtomicReference in 3.12 series. So, I didn't add it but I think we should discuss the structure of the org-website samples within the team and decide on a set of samples. We can add it afterward. What do you think about it?

Reply:
Yes, we can add those samples later, after an in-team discussion.